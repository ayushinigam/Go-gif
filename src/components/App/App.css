.appContainer {
  display: flex;
  flex-direction: column;
  padding: 30px 60px;
}

@keyframes App-logo-spin {
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(360deg);
  }
}
