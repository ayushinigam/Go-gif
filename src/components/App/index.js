import React from 'react';
import Search from '../Search';
import './App.css';

class App extends React.Component {
    state = {
        searchValue: '',
    }
    updateSearchValue = (value) => {
        this.setState({searchValue: value});
        console.log(this.state.searchValue, value);
        
    }
render() {
  return (
    <div className="appContainer">
      <Search updateSearchValue={this.updateSearchValue}/>
    </div>
  );
}
}

export default App;
