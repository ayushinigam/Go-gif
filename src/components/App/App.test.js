import React from 'react';
import renderer from 'react-test-renderer';
import {shallow, mount} from 'enzyme';
import App from './index';

describe(('App'), () => {
  it('renders correctly', () => {
    const tree = renderer.create(<App/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

