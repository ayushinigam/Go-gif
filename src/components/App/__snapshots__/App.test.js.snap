// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<div
  className="appContainer"
>
  <input
    className="searchInput"
    onBlur={[Function]}
    onChange={[Function]}
    onKeyDown={[Function]}
    placeholder="Search"
    type="text"
  />
</div>
`;
